Week2/todo_app/program.js

```diff
@@ -23,37 +23,98 @@ class TodoModel {
 
     create(description, callback) {
         // Write code and query to create a new TODO item
+
+        this.dbConnection.query('INSERT INTO todo_items (text, user_id) VALUES(?,?)', [description, 4], function (err, results, fields) {
```

Comment:
You can do this - but it's better to make your query string a variable instead of putting it directly there. Just a suggestion :) I see that you do it here because our queries are short, when you encounter huge queries in the future, better not do this.